packages/python-packages/doc-warden/warden/warden_common.py

```diff
@@ -184,29 +183,13 @@ def check_folder_against_exclusion_list(folder, path_exclusion_list):
 
     return os.path.normpath(folder) in path_exclusion_list
 
-# Get list of pom files as defined in the modules property in pom.client.xml in the java repo
-def get_defined_java_packages(configuration):
-    pom_locations = []
-    root = parse_pom(os.path.join(configuration.target_directory, 'pom.client.xml'))
-    modules = root.find('modules')
-
-    if modules is not None:
-        for module in modules:
-            pom_path = os.path.normpath(os.path.join(configuration.target_directory, module.text, 'pom.xml'))
-            if os.path.isfile(pom_path):
-                pom_locations.append(pom_path)
-
-    return pom_locations
-
-
-
 # given a pom.xml, crack it open and ensure that it is actually a package pom (versus a parent pom)
 def is_java_pom_package_pom(file_path):
     root = parse_pom(file_path)
-    jar_tag = root.find('packaging')
+    artifactIdTag = root.find('parent/artifactId')
 
-    if jar_tag is not None:
-        return jar_tag.text == 'jar'
+    if artifactIdTag is not None:
+        return artifactIdTag.text == 'azure-client-sdk-parent' or artifactIdTag.text == 'azure-data-sdk-parent'
```

Comment:
Changing away from `all poms with jar packaging` to `any pom that has a parent artifact id`. Am I missing anything?

Reply:
Well all the ones with artifactId set to `azure-client-sdk-parent` or `azure-data-sdk-parent`

Reply:
Fair. Alrighty.